parmed/__init__.py

```diff
@@ -6,7 +6,7 @@
 # Version format should be "major.minor.patch". For beta releases, attach
 # "-beta#" to the end. The beta number will be turned into another number in the
 # version tuple
-__version__ = '2.0.12'
+__version__ = '2.1.0'
```

Comment:
Exactly.  A small thing, like fixing a bug or adding a small feature is a bump from `x.y.z` to `x.y.z+1`.  Minor API changes, like what I did here, is `x.y.z` to `x.y+1.0`.  Major changes increment `x`.